### ISSUE TITLE
Prey 1.5.1 deb install fails on Ubuntu 16.04

### ISSUE THREAD

**Issue by kallisti5** [prey/prey-node-client#201]
```
chowning /usr/lib/prey/versions/1.5.1/node_modules/nodemailer/node_modules/buildmail to uid 999 and gid 999
chowning /usr/lib/prey/versions/1.5.1/node_modules/nodemailer/node_modules to uid 999 and gid 999
chowning /usr/lib/prey/versions/1.5.1/node_modules/nodemailer to uid 999 and gid 999
chowning /usr/lib/prey/versions/1.5.1/node_modules to uid 999 and gid 999
chowning /usr/lib/prey/versions/1.5.1 to uid 999 and gid 999
chowning /usr/lib/prey/versions to uid 999 and gid 999
chowning /usr/lib/prey to uid 999 and gid 999
Setting permissions on /etc/prey
chowning /etc/prey/prey.conf to uid 999 and gid 999
chowning /etc/prey to uid 999 and gid 999
Setting permissions on /var/log/prey.log
chowning /var/log/prey.log to uid 999 and gid 999
Running "config activate" as prey
--
Ensuring presence of config dir: /etc/prey
Syncing /etc/prey/prey.conf with /usr/lib/prey/versions/1.5.1/prey.conf.default
Setting up this as current...
Warning: This version is already set as current.
Setting permissions on /usr/lib/prey/current
Running post_activate hooks...
Voila! All good.
--
Installing init scripts.
dpkg: error processing package prey (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 prey
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by javo** [comment]
Hey @kallisti5,
We're trying to figure out what is causing this issue, that error is due to a dependencies problem.
For now i recommend you to purge prey:
```
sudo dpkg --purge prey
```
Then install the necessary dependencies
```
sudo apt-get install python-gtk2 scrot mpg123 streamer
```
And finally reinstall prey. Try that and please let me know how it went, it would help us a lot to figure out what's happening

Thanks!

**Comment by CandricS** [comment]
I'm having the same issue. However, those dependencies were already installed because when I run it I get the following:

~$ sudo apt-get install python-gtk2 scrot mpg123 streamer
Reading package lists... Done
Building dependency tree       
Reading state information... Done
python-gtk2 is already the newest version (2.24.0-4ubuntu1).
python-gtk2 set to manually installed.
mpg123 is already the newest version (1.22.4-1).
mpg123 set to manually installed.
scrot is already the newest version (0.8-17).
scrot set to manually installed.
streamer is already the newest version (3.103-3build1).
streamer set to manually installed.

**Comment by noex869** [comment]
Looks like the following line is failing causing the error code (1)
`DISPLAY=:0.0 su $PREY_USER -c "$PREY_BIN config gui --check-file $TEMP_OLD_CONFIG"`

Setting the shell(because prey's default is /bin/false) prevents the error but does not launch the gui
`DISPLAY=:0.0 su $PREY_USER -s /bin/bash -c "$PREY_BIN config gui --check-file $TEMP_OLD_CONFIG"`

For now I just commented out that line and ran `/usr/lib/prey/current/bin/prey config gui` after the install

How to edit a deb package: http://unix.stackexchange.com/questions/138188/easily-unpack-deb-edit-postinst-and-repack-deb/138190

**Comment by bhovell2** [comment]
Having recently given up on prey for Ubuntu, I continue to wonder at comments by the developers which suggest they've never actually installed prey on a native Ubuntu box (or VM) and tested it themselves. Having to continuously ask users to offer suggested solutions to the same problems repeatedly reported from version to version is odd enough.

But even stranger is that fixes suggested to solve problems in the ever-broken post-install script obviously never make it into the most recent packages. So what exactly is the point of asking for fixes?

**Comment by raliste** [comment]
The point is that those fixes should make it to the latest package.

The real issue here, as I see it, is velocity. We are not being able to answer nor fix issues at the expectations that I think all of us share. I would like to see us, the core developers, answering and fixing things promptly, but I've to use the "we are small team" card as we are working on a lot of things at the same time. I can only assure that this *will* get fixed, and pretty soon. Our streak was amazingly good a few months ago, and I'm pretty sure we'll be able to get back on track, that's why it's so important for us that you guys keep posting issues and giving us feedback, it reminds us that we have to put focus on the community first, before our own internal things.

We gather as a team every two weeks to discuss the top issues for the following two weeks of work. I'll put these issues first so that we can hopefully release version 1.6.1 by the end of this cycle.

**Comment by bhovell2** [comment]
I really don't know why I keep fooling with this except it's frustrating that every version seems to break in just one slight way or another on recent ubuntu versions.

Since I discovered there is now an "official" version of prey in the ubuntu repository, I decided to try that one.  Hope springs eternal.

I followed the instructions above to completely purge, reload dependencies, then reinstall prey, Amazingly enough, that all went flawlessly. Or so it seemed after running the setup script, but -- alas! -- it was not to be.

This time, when running the prey.sh script, I get this output:

////////////////////// BEGIN OUTPUT/////////////////

 ## PREY 0.6.2 spreads its wings!
 ## Linux rex 4.4.0-22-generic #40-Ubuntu SMP Thu May 12 22:03:46 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
 ## Fri May 27 23:34:53 COT 2016

 -- Looking for connection...
 -- Got network connection!

 >> Registering device under account!

 -- Gathering system and hardware information...
 -- Sending request to Control Panel...
 -- Got response status 502.
 -- Couldn't add this device to your account. Make sure you have available slots!

///////////// END OUTPUT///////////

There were originally three devices registered to this account but I had to remove the entry for the Ubuntu laptop when I couldn't get a clean install of any prey version after upgrading to 16.04.

Therefore only two devices are currently registered to that account. There are three slots. Reason would suggest the remaining one should be "available". But that's obviously wrong.

I even removed an additional device from the control panel (leaving only one registered), but that made no difference. The laptop still wouldn't register. But the other device I removed (a smartphone) re-registered with no problem.

So this round of failure apparently owes to something on the server side rather than on the client. So that's progress of a sort. But still no joy.

**Comment by raliste** [comment]
New version 1.6.1 should fix this issue. We would really appreciate your test results with the new installer available at https://github.com/prey/prey-node-client/releases/tag/v1.6.1

**Comment by CandricS** [comment]
Version 1.6.1 seems to work fine. Thanks.

I'll report if I come across any issues.

**Comment by bhovell2** [comment]
Hi, Rodrigo ...

OK, this version 1.6.1 appears to have installed on Ubuntu without
failure.

However, it's been awhile since I last installed this on Ubuntu and
there are no visible instructions how to actually get the setup
completed. I remain a bit astonished how much the Prey version for
Ubuntu seems like a kit car delivered without assembly instructions.
Generally they would be mentioned in the README file if nowhere
else. But there's not a word.

I have the API defined in the new prey.conf file but have no idea
what comes next to make connection and retrieve the device id. Can
you offer a hint?

Best,

Bud Hovell

**Comment by raliste** [comment]
@bhovell2 after the install process a GUI for configuration should pop up. If that wasn't the case you could try running `prey config account setup` with sudo, which will guide you through the process of attaching your device to your account.

**Comment by bhovell2** [comment]
Hi, Rodrigo ....
>
> @bhovell2 <https://github.com/bhovell2> after the install process
> a GUI for configuration should pop up. If that wasn't the case you
> could try running |prey config account setup| with sudo, which
> will guide you through the process of attaching your device to
> your account.
>

You might want to mention this in the README. I've never seen a pop
up in this (or I think any pervious) version.

Also, this is the ouput I get:

> *[rex]bud   > sudo ./prey config account setup**
> **[sudo] password for bud: **
> **Error! Account already set up! Run with -f/--force to continue
> anyway.**
> * *
> **[rex]bud   > sudo ./prey -f config account setup**
> * *
> **  ████████████████ ████████████████ ████████████████
> ██            ██**
> **  ██            ██ ██               ██            ██
> ██            ██**
> **  ██            ██ ██               ████████████████
> ██            ██**
> **  ██            ██ ██               ██              
> ██            ██**
> **  ████████████████ ██               ████████████████
> ████████████████**
> ** 
> ██                                                               ██**
> **  ██                                                
> ████████████████**
> * *
> **  PREY 1.6.1 spreads its wings!**
> **  Current time: Tue Jun 21 2016 18:36:01 GMT-0500 (COT)**
> **  Running with PID 8939 as root over Node.js v4.4.5 on a x64
> linux system (undefined)**
> * *
> **    error Tue, 21 Jun 2016 23:36:11 GMT Error: socket hang up**
> **     info Tue, 21 Jun 2016 23:36:11 GMT [control-panel]
> Gathering device data...**
> **     info Tue, 21 Jun 2016 23:36:11 GMT [reports] Gathering
> specs report.**
> **    error Tue, 21 Jun 2016 23:36:11 GMT [reports] Unable to get
> storage_devices_list: TODO!**
> **     info Tue, 21 Jun 2016 23:36:11 GMT [reports] Report
> gathered: specs**
> **    error Tue, 21 Jun 2016 23:36:19 GMT [control-panel] Unable
> to register device: Invalid credentials.**
> **    error Tue, 21 Jun 2016 23:36:19 GMT Error: Invalid
> credentials.**
> **     info Tue, 21 Jun 2016 23:36:19 GMT 1 plugins loaded with 1
> errors.**
> ** critical Tue, 21 Jun 2016 23:36:19 GMT UNCAUGHT EXCEPTION:
> Invalid credentials.**
> **     info Tue, 21 Jun 2016 23:36:19 GMT Gracefully shutting
> down. Will self-destruct in 5 secs.**
> **     info Tue, 21 Jun 2016 23:36:19 GMT [hooks] Unregistering
> hooks.**
> **     info Tue, 21 Jun 2016 23:36:24 GMT Time's up, shutdown
> forced. Have a jolly good day sir.*

Is that what's expected?

Also, there's nothing mentioned in the README about the necessity to
manually configure the file /etc/prey/prey.conf

Is that supposed to happen automagically also?  I did add my API, so
this is what's now in that file:

> *#############################
> # Prey Configuration
> #############################
>
> # List of plugins to load at startup. For options, run 'prey
> config plugins'.
> plugin_list = control-panel
>
> # Whether to try to connect to an open Wifi network if not online,
> when needing to send data.
> auto_connect = false
>
> # Allows Prey to auto-update itself when new versions are released.
> auto_update = true
>
> # Fetch edge releases before they reach the stable branch. No
> effect without auto_update.
> download_edge = false
>
> # Whether to inform about runtime errors to the development team.
> It helps. :)
> send_crash_reports = true
>
> # HTTP proxy server to try if a request fails (eg.
> http://my.proxy.com:3128)
> try_proxy =
>
> # These are temporary placeholders to persist the values when
> upgrading. Do not remove.
> api_key =
> device_key =
>
> [control-panel]
>
> # Host to connect to.
> host = solid.preyproject.com
>
> # Protocol to use. Either http or https.
> protocol = https
>
> # Your account's API key.
> api_key = '147db1401519'
>
> # Your device's unique key.
> device_key =
>
> # Includes status info (battery info, logged user, etc) when
> checking in.
> send_status_info = true
>
> # Whether to send hardware information on boot, to detect any changes.
> scan_hardware = false*

I do not, of course, know what the device key is, so couldn't add that.

Best,

Bud

**Comment by raliste** [comment]
```
 *[rex]bud   > sudo ./prey config account setup**
 **[sudo] password for bud: **
 **Error! Account already set up! Run with -f/--force to continue
 anyway.**
 * *
 **[rex]bud   > sudo ./prey -f config account setup**```

You have to add the `-f` at the end, as in `sudo ./prey config account setup -f`. (I know, we have semantic issues with our CLI).

Also, it's not required for you to edit the prey.conf manually. Both the CLI and the GUI should update it for you.

Please do let me know if you are able to run `sudo ./prey config account setup -f` successfully.

**Comment by bhovell2** [comment]
|

Hi, Rodrigo ...

|
> ||You have to add the `-f` at the end, as in `sudo ./prey config
> account setup -f`. (I know, we have semantic issues with our CLI).
> Also, it's not required for you to edit the prey.conf manually.
> Both the CLI and the GUI should update it for you. Please do let
> me know if you are able to run `sudo ./prey config account setup
> -f` successfully.||
|||
IMO, you should to explicitly provide all these manual install
instructions in the READEME (at least for Ubuntu). Relying on
automation alone is probably going to create a lot of confusion for
people. And that only assumes they know where to go look.

That said, running the command you provided did get the device
registered on the website.

||||However, when I tried to update the location of the device from
the website, it failed with a message saying it couldn't get the
location "in time" (presumably timed out).||

So I ran the debug  "prey -D" with this log output:

> PREY 1.6.1 spreads its wings!
>   Current time: Tue Jun 21 2016 19:52:16 GMT-0500 (COT)
>   Running with PID 10236 as root over Node.js v4.4.5 on a x64
> linux system (undefined)
>
>     debug Wed, 22 Jun 2016 00:52:16 GMT [updater] Checking for
> updates on stable branch...
>     debug Wed, 22 Jun 2016 00:52:17 GMT Command received:
> {"command":"start","target":"geofencing"}
>     debug Wed, 22 Jun 2016 00:52:17 GMT [hooks] Hook triggered:
> command
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Starting action:
> geofencing
>     debug Wed, 22 Jun 2016 00:52:17 GMT [control-panel] Sending
> GET request #1 to https://solid.preyproject.com
>     debug Wed, 22 Jun 2016 00:52:17 GMT [hooks] Hook triggered: action
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Running: geofencing
>     debug Wed, 22 Jun 2016 00:52:17 GMT [control-panel] Sending
> GET request #1 to https://solid.preyproject.com
>      info Wed, 22 Jun 2016 00:52:17 GMT Plugin loaded: control-panel
>      info Wed, 22 Jun 2016 00:52:17 GMT 1 plugins loaded with 0
> errors.
>      info Wed, 22 Jun 2016 00:52:17 GMT Initialized.
>      info Wed, 22 Jun 2016 00:52:17 GMT [triggers] Watching:
> connection, network, power
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Starting
> trigger: connection
>     debug Wed, 22 Jun 2016 00:52:17 GMT [network] Getting
> connection status
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Running: connection
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Starting
> trigger: network
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Starting
> trigger: power
>     debug Wed, 22 Jun 2016 00:52:17 GMT [providers] Fetching
> active_access_point_name
>     debug Wed, 22 Jun 2016 00:52:17 GMT [providers] Fetching
> private_ip
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Running: network
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Running: power
>     debug Wed, 22 Jun 2016 00:52:17 GMT [hooks] Hook triggered:
> connected
>     debug Wed, 22 Jun 2016 00:52:17 GMT [long-polling] Fetching
> instructions...
>     debug Wed, 22 Jun 2016 00:52:17 GMT [control-panel] Attempt #1
> took 0.496 seconds.
>      info Wed, 22 Jun 2016 00:52:17 GMT [actions] Stopped: geofencing
>     debug Wed, 22 Jun 2016 00:52:17 GMT [control-panel] Attempt #1
> took 0.51 seconds.
>     debug Wed, 22 Jun 2016 00:52:17 GMT [control-panel] Syncing
> settings.
>     debug Wed, 22 Jun 2016 00:52:18 GMT [hooks] Hook triggered: action
>     debug Wed, 22 Jun 2016 00:52:18 GMT Removing command from DB:
> start-geofencing
>     debug Wed, 22 Jun 2016 00:52:32 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:52:47 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:53:02 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:53:17 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:53:18 GMT [long-polling]
> Re-scheduling request
>     debug Wed, 22 Jun 2016 00:53:18 GMT [long-polling] Fetching
> instructions...
>     debug Wed, 22 Jun 2016 00:53:18 GMT [long-polling] Request  ended
>     debug Wed, 22 Jun 2016 00:53:32 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:53:42 GMT [long-polling]
> Re-scheduling request
>     debug Wed, 22 Jun 2016 00:53:42 GMT [long-polling] Fetching
> instructions...
>      info Wed, 22 Jun 2016 00:53:42 GMT [long-polling] Got 1 commands.
>     debug Wed, 22 Jun 2016 00:53:42 GMT Command received:
> {"command":"get","target":"location"}
>     debug Wed, 22 Jun 2016 00:53:42 GMT [hooks] Hook triggered:
> command
>     debug Wed, 22 Jun 2016 00:53:42 GMT [providers] Fetching location
>     debug Wed, 22 Jun 2016 00:53:42 GMT [geo] Getting location via
> native geoloc
>     debug Wed, 22 Jun 2016 00:53:42 GMT [long-polling] Request  ended
>     debug Wed, 22 Jun 2016 00:53:42 GMT [geo] Error getting
> location using native strategy: Error: Command failed: /bin/sh -c
> dbus-send --print-reply
> --dest=org.freedesktop.Geoclue.Providers.UbuntuGeoIP
> /org/freedesktop/Geoclue/Providers/UbuntuGeoIP
> org.freedesktop.Geoclue.Position.GetPosition
> Failed to open connection to "session" message bus: Did not
> receive a reply. Possible causes include: the remote application
> did not send a reply, the message bus security policy blocked the
> reply, the reply timeout expired, or the network connection was
> broken.
>
>     debug Wed, 22 Jun 2016 00:53:42 GMT [geo] Getting location via
> google api
>     debug Wed, 22 Jun 2016 00:53:42 GMT [providers] Fetching
> access_points_list
>     debug Wed, 22 Jun 2016 00:53:47 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:53:56 GMT [geo] Sending AP data to
> Google API
>     debug Wed, 22 Jun 2016 00:53:57 GMT [geo] Processing Google
> API response
>     debug Wed, 22 Jun 2016 00:53:57 GMT [geo] Got location via
> Google API
>     debug Wed, 22 Jun 2016 00:53:57 GMT [hooks] Hook triggered: data
>      info Wed, 22 Jun 2016 00:53:57 GMT [reports] Gathering status
> report.
>     debug Wed, 22 Jun 2016 00:53:57 GMT [providers] Fetching uptime
>     debug Wed, 22 Jun 2016 00:53:57 GMT [reports] [3/4] Got uptime
>     debug Wed, 22 Jun 2016 00:53:57 GMT [providers] Fetching
> logged_user
>     debug Wed, 22 Jun 2016 00:53:57 GMT [providers] Fetching
> active_access_point
>     debug Wed, 22 Jun 2016 00:53:57 GMT [providers] Fetching
> battery_status
>     debug Wed, 22 Jun 2016 00:53:57 GMT [reports] [2/4] Got
> logged_user
>     debug Wed, 22 Jun 2016 00:53:58 GMT [reports] [1/4] Got
> battery_status
>     debug Wed, 22 Jun 2016 00:54:02 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:54:12 GMT [reports] [0/4] Got
> active_access_point
>      info Wed, 22 Jun 2016 00:54:12 GMT [reports] Report gathered:
> status
>      info Wed, 22 Jun 2016 00:54:12 GMT [control-panel] Posting data
>     debug Wed, 22 Jun 2016 00:54:12 GMT [control-panel] Sending
> POST request #1 to https://solid.preyproject.com
>     debug Wed, 22 Jun 2016 00:54:12 GMT [control-panel] Attempt #1
> took 0.451 seconds.
>      info Wed, 22 Jun 2016 00:54:12 GMT [control-panel] Got 200
> response: OK
>     debug Wed, 22 Jun 2016 00:54:17 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:54:32 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:54:42 GMT [long-polling]
> Re-scheduling request
>     debug Wed, 22 Jun 2016 00:54:42 GMT [long-polling] Fetching
> instructions...
>     debug Wed, 22 Jun 2016 00:54:42 GMT [long-polling] Request  ended
>     debug Wed, 22 Jun 2016 00:54:47 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:55:02 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:55:17 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:55:32 GMT [network] Getting
> connection status
>     debug Wed, 22 Jun 2016 00:55:42 GMT [long-polling]
> Re-scheduling request
>     debug Wed, 22 Jun 2016 00:55:42 GMT [long-polling] Fetching
> instructions...
>     debug Wed, 22 Jun 2016 00:55:42 GMT [long-polling] Request  ended
>     debug Wed, 22 Jun 2016 00:55:47 GMT [network] Getting
> connection status
> ^C     warn Wed, 22 Jun 2016 00:55:52 GMT Got INT signal.
>      info Wed, 22 Jun 2016 00:55:52 GMT Gracefully shutting down.
> Will self-destruct in 5 secs.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unloading plugins.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unloading control-panel
> plugin...
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Clearing
> current request
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Aborting
> current request
>     debug Wed, 22 Jun 2016 00:55:52 GMT Plugin unloaded: control-panel
>     debug Wed, 22 Jun 2016 00:55:52 GMT Stopping actions.
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: connection
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: network
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: power
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unloading hooks.
>      info Wed, 22 Jun 2016 00:55:52 GMT [hooks] Unregistering hooks.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Canceling reports.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unwatching triggers.
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: connection
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: network
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: power
>     debug Wed, 22 Jun 2016 00:55:52 GMT Cleaning up temporary files.
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Request aborted
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Request  ended
I then hit Control-C to end the session
>      info Wed, 22 Jun 2016 00:55:52 GMT Shutdown complete. Have a
> jolly good day sir.
||||
Then ran prey --run 'get location' at the command line and got this
output:

|
> |
> location { lat: 6.205090299999999,
>   lng: -75.57222469999999,
>   accuracy: 30,
>   method: 'wifi' }|

|So prey can locally discover the location coordinates but cannot
retrieve it by request from the website.

Best,

Bud Hovell
||





|

**Comment by bhovell2** [comment]
My mistake, Rodrigo ...

I should have shown:

[....]

>     debug Wed, 22 Jun 2016 00:55:42 GMT [long-polling] Request  ended
>     debug Wed, 22 Jun 2016 00:55:47 GMT [network] Getting
> connection status
I then hit Control-C to end the session
> ^C     warn Wed, 22 Jun 2016 00:55:52 GMT Got INT signal.
>      info Wed, 22 Jun 2016 00:55:52 GMT Gracefully shutting down.
> Will self-destruct in 5 secs.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unloading plugins.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unloading control-panel
> plugin...
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Clearing
> current request
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Aborting
> current request
>     debug Wed, 22 Jun 2016 00:55:52 GMT Plugin unloaded: control-panel
>     debug Wed, 22 Jun 2016 00:55:52 GMT Stopping actions.
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: connection
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: network
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: power
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unloading hooks.
>      info Wed, 22 Jun 2016 00:55:52 GMT [hooks] Unregistering hooks.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Canceling reports.
>     debug Wed, 22 Jun 2016 00:55:52 GMT Unwatching triggers.
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: connection
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: network
>      info Wed, 22 Jun 2016 00:55:52 GMT [actions] Stopping: power
>     debug Wed, 22 Jun 2016 00:55:52 GMT Cleaning up temporary files.
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Request aborted
>     debug Wed, 22 Jun 2016 00:55:52 GMT [long-polling] Request  ended
>      info Wed, 22 Jun 2016 00:55:52 GMT Shutdown complete. Have a
> jolly good day sir.

**Comment by bhovell2** [comment]
Hi, Rodrigo ...

After fussing around a bit with this installation, I put in a cron
job to run prey every 20 minutes and direct output to the
/var/log/prey.log file. Now tested and everything is apparently
working flawlessly when I set the device to "lost" in the dashboard.

Good job!  And thanks for getting that fix in place.
-- 
*Bud Hovell*

If you aren't taking flak, you aren't over the target.

**Comment by javo** [comment]
This issue is solved installing 1.6.1 client version. 😄 https://preyproject.com/download
